[2022-12-11 19:18:35,225] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl_DAG.run_etl_process manual__2022-12-11T19:18:13.995964+00:00 [queued]>
[2022-12-11 19:18:35,356] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl_DAG.run_etl_process manual__2022-12-11T19:18:13.995964+00:00 [queued]>
[2022-12-11 19:18:35,374] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-11 19:18:35,376] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-12-11 19:18:35,387] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-11 19:18:35,587] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): run_etl_process> on 2022-12-11 19:18:13.995964+00:00
[2022-12-11 19:18:35,668] {standard_task_runner.py:52} INFO - Started process 174 to run task
[2022-12-11 19:18:35,765] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'etl_DAG', 'run_etl_process', 'manual__2022-12-11T19:18:13.995964+00:00', '--job-id', '192', '--raw', '--subdir', 'DAGS_FOLDER/etl_DAG.py', '--cfg-path', '/tmp/tmpmkwtvl_i', '--error-file', '/tmp/tmp7ftzj1d5']
[2022-12-11 19:18:35,796] {standard_task_runner.py:80} INFO - Job 192: Subtask run_etl_process
[2022-12-11 19:18:36,713] {task_command.py:369} INFO - Running <TaskInstance: etl_DAG.run_etl_process manual__2022-12-11T19:18:13.995964+00:00 [running]> on host 0163e0790208
[2022-12-11 19:18:38,488] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=etl_DAG
AIRFLOW_CTX_TASK_ID=run_etl_process
AIRFLOW_CTX_EXECUTION_DATE=2022-12-11T19:18:13.995964+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-11T19:18:13.995964+00:00
[2022-12-11 19:19:34,160] {python.py:173} INFO - Done. Returned value was: None
[2022-12-11 19:19:35,468] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=etl_DAG, task_id=run_etl_process, execution_date=20221211T191813, start_date=20221211T191835, end_date=20221211T191935
[2022-12-11 19:19:36,501] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-11 19:19:37,075] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
